[2023-09-13T07:38:27.845+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2023-09-13T07:38:27.867+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2023-09-13T07:38:27.868+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:27.924+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-23 00:00:00+00:00
[2023-09-13T07:38:27.937+0000] {standard_task_runner.py:57} INFO - Started process 87 to run task
[2023-09-13T07:38:28.002+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-23T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpm6a3ocg6']
[2023-09-13T07:38:28.006+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask extract_data
[2023-09-13T07:38:28.281+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-23T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:28.592+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-23T00:00:00+00:00'
[2023-09-13T07:38:29.005+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:29.056+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230823T000000, start_date=20230913T073827, end_date=20230913T073829
[2023-09-13T07:38:29.166+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:29.253+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:08.558+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2023-09-15T11:31:08.566+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2023-09-15T11:31:08.568+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:08.582+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-23 00:00:00+00:00
[2023-09-15T11:31:08.596+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-23T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpopl0odsk']
[2023-09-15T11:31:08.599+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask extract_data
[2023-09-15T11:31:08.591+0000] {standard_task_runner.py:57} INFO - Started process 333 to run task
[2023-09-15T11:31:08.701+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-23T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:08.798+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-23T00:00:00+00:00'
[2023-09-15T11:31:09.082+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:09.095+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230823T000000, start_date=20230915T113108, end_date=20230915T113109
[2023-09-15T11:31:09.141+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:09.161+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
